server/services/incomingMovementsService.ts

```diff
@@ -9,9 +10,29 @@ export default class IncomingMovementsService {
     private readonly welcomeApiFactory: (token: string) => WelcomeApi
   ) {}
 
+  private moveTypeGroupingFunction(item: Movement): string {
```

Comment:
rather than string might be nicer to have an enum to represent this set of closed values

Also might be nice the represent the null explicitly - either as part of the return value or even an explicit enum value `OTHER` or whatever

(note upper case to represent that these are constants)

Reply:
(Using an explicit value rather than `null` is an example of the [https://en.wikipedia.org/wiki/Null_object_pattern](Null object pattern))